spec/newgistics/update_shipment_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+
+RSpec.describe Newgistics::UpdateShipment do
+  include IntegrationHelpers
+
+  describe '#update' do
+    before { use_valid_api_key }
+
+    vcr_options = { cassette_name: 'update_shipment_contents/update/success' }
+    context "when the shipment contents are changed successfully", vcr: vcr_options do
+      it "returns true" do
+        update_shipment = described_class.new(updated_attributes)
+
+        success = update_shipment.update
+
+        expect(success).to eq true
+      end
+
+      it 'sets success' do
+        update_shipment = described_class.new(updated_attributes)
+
+        update_shipment.update
+
+        expect(update_shipment.success).to eq true
+      end
+    end
+
+    vcr_options = { cassette_name: 'update_shipment_contents/update/failure' }
+    context "when the updating the shipment contents fails", vcr: vcr_options do
+      it 'updates the inbound_return object with the errors' do
+        update_shipment = described_class.new(
+          updated_attributes(shipment_id: 'INVALID')
+        )
+
+        success = update_shipment.update
+
+        expect(success).to eq false
+        expect(update_shipment.errors.size).to eq 1
+        expect(update_shipment.errors.first).to eq(
+          "Invalid shipment ID."
+        )
+      end
+    end
+  end
+
+  describe "#success?" do
```

Comment:
I think we should de-couple this from the `#update` method, I'd say just test for the return values depending on what the `success` attribute has:

```ruby
it "returns true if success is true"

it "returns false if success is false"

it "returns false if success is nil"
```